stock_operating_unit/model/stock.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp.tools.translate import _
+from openerp import api, fields, models
+from openerp.exceptions import Warning as UserError
+
+
+class StockWarehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    operating_unit_id = fields.Many2one('operating.unit',
+                                        'Operating Unit',
+                                        default=lambda self:
+                                        self.env['res.users'].
+                                        operating_unit_default_get(self._uid))
```

Comment:
`self.env.uid` is a cleaner (no access to a "private" attribute).

You would have less line breaks with such indentation:

```
    operating_unit_id = fields.Many2one(
        comodel_name='operating.unit',
        string='Operating Unit',
        default=lambda self: self.env['res.users'].
                operating_unit_default_get(self.env.uid))
    )
```